### PR TITLE
Update fhvp.json

### PR DESCRIPTION
**Pull request by TraXX0r**
Fixed typos

### CODE REVIEW

**Comment by zzorba** [comment]
My assumptions j was that like treacheries, the parentheses are the sub name of events (like Yuggoth).

Possible we need a tweak of the code to render them as such

**Comment by zzorba** [comment]
The bonded behavior might also break if name doesn't match. It should probably remain as a subname

**Comment by TraXX0r** [comment]
Aetheric Current is back to how it was before due to zzorbas comment

**Comment by zzorba** [comment]
Sometimes there are multiple versions of a card that it is bonded to, not just one -- i.e. there are two hallowed mirrors, but only one set of Soothing Melodies.

File: pack/fhv/fhvp.json [hunk 1]
```diff
@@ -920,7 +923,7 @@
     "quantity": 2,
     "skill_intellect": 1,
     "skill_willpower": 1,
-    "text": "Fast. Play after an investigator reveals a h token during a skill test at your location.\nDiscover 1 clue at your location.",
+    "text": "Fast. Play after an investigator reveals a [cursed] token during a skill test at your location.\nDiscover 1 clue at your location.",
```
Comment (zzorba) [inline pack/fhv/fhvp.json:926]: This should be [curse] not [cursed]

Comment (TraXX0r) [reply pack/fhv/fhvp.json]: Fixed

**Comment by TraXX0r** [comment]
@5argon Added "bounded_count" to all bonded cards where it was missing (that is Uncanny Groth in case of the Myconids and not the Myconids itself, at least I think that is where it belongs) although I don't know what "bonded_count" is even used for?! E.g. SCED for TableTopSimulator just uses "quantity" to determine the number of copies to spaw when you load a deck.

Another thing I just noticed: For all older cards (see e.g. Pendant of the Queen / Segment of Onyx) both the parent and the bonded card show each other listed under Bonded Cards, while for the new cards only the parent shows the bonded cards (but not vice versa). In both cases the code only has "bonded_to" set for the bonded card to reference the parent but not vice versa. Someone know why is that?

**Comment by 5argon** [comment]
> @5argon Added "bounded_count" to all bonded cards where it was missing (that is Uncanny Groth in case of the Myconids and not the Myconids itself, at least I think that is where it belongs) although I don't know what "bonded_count" is even used for?! E.g. SCED for TableTopSimulator just uses "quantity" to determine the number of copies to spaw when you load a deck.
> 
> Another thing I just noticed: For all older cards (see e.g. Pendant of the Queen / Segment of Onyx) both the parent and the bonded card show each other listed under Bonded Cards, while for the new cards only the parent shows the bonded cards (but not vice versa). In both cases the code only has "bonded_to" set for the bonded card to reference the parent but not vice versa. Someone know why is that?

Oh right, it should be on the Uncanny Growth, my bad.

I guess `bonded_cards` is meant for the ability to put the listed cards into play (therefore both Pendant of the Queen / Segment of Onyx both has it because they can change back and forth) and  `bonded_to` is meant for "Bonded (___)" in the text box (therefore it is one way). (?)

**Comment by 5argon** [comment]
Also I wonder if an entire Bonded Hank (#16) card having `hidden` is alright? I am guessing this is a workaround so that his resolute version doesn't show up in the deck creator (?), but now it is also missing from card listing completely.

image

**Comment by TraXX0r** [comment]
Changed bonded Hank back to the original ("code": "10016a")

**Comment by TraXX0r** [comment]
Fixed deck_limit of ugraded Rod of Carnamagos from 1 to 2

**Comment by zzorba** [comment]
All the Rot cards are missing the '.' at the end of Bonded (Rod of Carnamagos).  Might break some peoples regexes (like it did mine).

**Comment by TraXX0r** [comment]
> All the Rot cards are missing the '.' at the end of Bonded (Rod of Carnamagos). Might break some peoples regexes (like it did mine).

Fixed

**Comment by TraXX0r** [comment]
Added "tags": "hd.hh" to the healing Ravenous Myconid, assuming that's the right way to give Carolyn and Vincent access to it?!

**Comment by zzorba** [comment]
Correct!

**Comment by zzorba** [comment]
I think it should be hd.hh. though (two periods)?

**Comment by zzorba** [comment]
@TraXX0r small typo on Alessandra's flavor, line 185.  Banquest should be banquet

**Comment by mikecheb** [comment]
On line 126, `"Is this the same evergy harnessed by Nikola Tesla?"`, "evergy" should be "energy"

**Pull request merged**